cells/addf/definition.json

```diff
@@ -0,0 +1,48 @@
+{
+    "description": "1 bit Full Adder",
+    "file_prefix": "gf180mcu_fd_sc_mcu9t5v0__addf",
+    "library": "gf180mcu_fd_sc_mcu9t5v0__addf",
+    "name": "addf",
+    "parameters": [],
+    "ports": [
+        [
+            "signal",
+            "A",
+            "input"
+        ],
+        [
+            "signal",
+            "B",
+            "input"
+        ],
+        [
+            "signal",
+            "CI",
+            "input"
+        ],
+        [
+            "signal",
+            "CO",
+            "output"
+        ],
+        [
+            "signal",
+            "S",
+            "output"
+        ],
+        [
+            "power",
+            "VDD",
+            "inout",
```

Comment:
These should be inputs (needs power) or outputs (supplies powers).

Reply:
I thought from [skywater](https://github.com/google/skywater-pdk-libs-sky130_fd_sc_hd/blob/ac7fb61f06e6470b94e8afdf7c25268f62fbd7b1/cells/ha/definition.json) files that it should be the same as the verilog files. Now I understand thank you.